client/src/theme/Components/reader/_return-menu.scss

```diff
@@ -58,6 +58,8 @@
       margin-bottom: -4px;
       font-size: 0.692em;
       color: $neutral50;
+      overflow-wrap: break-word;
```

Comment:
Looks good! Max, can you put a comment here saying that we're using both attributes to support older browsers?
You can site this too.